.github/ISSUE_TEMPLATE/------suggest-a-development-implementation-improvement-.md

```diff
@@ -2,7 +2,7 @@
 name: "\U0001F9D1‍\U0001F4BB Suggest a Development/implementation improvement "
 about: Suggest something for the UI Kit or website code.
 title: ''
-labels: ''
+labels: bug
```

Comment:
is "bug" the right label for this type? should it be "enhancement"?

Reply:
This was meant to be on the bug ticket. I will add enhancement to this template and bug to our bug template.